corehq/apps/userreports/ui/forms.py

```diff
@@ -107,6 +107,14 @@ def __init__(self, domain, *args, **kwargs):
         self.domain = domain
         super(ConfigurableDataSourceEditForm, self).__init__(*args, **kwargs)
 
+        from corehq import toggles
```

Comment:
Does this have a circular import? I think its more pythonic to import things at the top of the file if not.